main/webapp/modules/core/scripts/project.js

```diff
@@ -63,7 +63,11 @@ Refine.reportException = function(e) {
 };
 
 function resize() {
-  var leftPanelWidth = 300;
+  var leftPanelWidth = JSON.parse(Refine.getPreference("ui.toolPanelWidth", 300));
```

Comment:
Perhaps the preference key could be improved: I would not call this a "tool panel". Perhaps just `ui.leftPanelWidth`?

Reply:
Yes, this pref name could be better but what if we have another preference to put tools panel on the right instead of the left?

Regards,
   Antoine

Reply:
What I am worried about with `tools` is that we never refer to this panel as being a tools panel anywhere. `ui.facetsHistoryPanelWidth` if you prefer?

Reply:
@wetneb : I'm totally open to another name. `ui.facetsHistoryPanelWidth` is good, but `ui.facetsPanelWidth` might be even simplier!

Regards, A.